client/client.go

```diff
@@ -26,8 +26,8 @@ type Client struct {
 	addr string
 	// basePath holds the path to prepend to the requests.
 	basePath string
-	// transport is the interface to send request with, it implements transport.Client.
-	transport transport.Client
+	// client used to send and receive http requests.
```

Comment:
"client used to send and recieve http requests" is valid and meaningful. No need to use passive voice in this case.